src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -22,13 +22,13 @@ public LibGpiodDriver()
             SafeChipIteratorHandle iterator = Interop.GetChipIterator();
             if (iterator == null)
             {
-                throw new IOException($"Unable to find a chip iterator, error code: {Marshal.GetLastWin32Error()}");
+                ThrowHelper.ThrowIOException(ThrowHelper.ExceptionResource.NoChipIteratorFound, Marshal.GetLastWin32Error());
             }
 
             _chip = Interop.GetNextChipFromChipIterator(iterator);
             if (_chip == null)
             {
-                throw new IOException($"Unable to find a chip, error code: {Marshal.GetLastWin32Error()}");
+                ThrowHelper.ThrowIOException(ThrowHelper.ExceptionResource.NoChipFound, Marshal.GetLastWin32Error());
```

Comment:
You should unify all to either use `throw ThrowHelper.GetIOException` or `ThrowHelper.ThrowIOException)` - I'm seeing both patterns in this PR. Btw, better name for the class could be simply `Error`

Reply:
followed ThrowHelper class in corefx which has only ThrowHelper.ThrowIOException(), but it was not good for using for methods need to return a value, so added ThrowHelper.GetIOException(), but yeah better to use only use ThrowHelper.GetIOException().

For the class name though I don't prefer renaming into Error, it is helper class so I would keep word Helper, but as am gonna use ThrowHelper.GetIOException() pattern instead of ThrowHelper.ThrowIOException() pattern, maybe rename it ExceptionHelper or ErrorHelper